src/test/java/io/github/guacsec/trustifyda/integration/HtmlReportTest.java

```diff
@@ -262,11 +262,6 @@ public void testBatchHtmlWithToken() throws IOException {
     // Verify multi tab layout
     List<HtmlElement> sectionElements = rootElement.getByXPath("./section");
     assertEquals(1, sectionElements.size());
-    List<HtmlAnchor> anchorElements =
```

Comment:
can you adapt this test by adding the imageRecommend / imageRemediationLink attributes to the configuration?

Reply:
will investigate.